Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -139,7 +139,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($token->equals(',')) {
+            if ($token->equals(',') && !in_array($tokens[$index - 1]->getContent(), array(')', ']'), true)) {
```

Comment:
Don't know about this method, will fix.

BTW, phpdoc should be improved to have better completion.

e.g.: `Tokens|Token[]`.